src/controllers/itemDetails/index.js

```diff
@@ -826,7 +826,7 @@ function setInitialCollapsibleState(page, item, apiClient, context, user) {
 
     renderScenes(page, item);
 
-    if (item.SpecialFeatureCount > 0) {
+    if (item.SpecialFeatureCount > 0 || item.Type == 'MusicArtist') {
```

Comment:
This should probably be ===
```suggestion
    if (item.SpecialFeatureCount > 0 || item.Type === 'MusicArtist') {
```